package.json

```diff
@@ -41,6 +41,7 @@
     "@sablier/indexers": "1.1.0",
     "@vercel/analytics": "^1.5.0",
     "axios": "^1.11.0",
+    "docusaurus-plugin-llms": "^0.2",
```

Comment:
Let's use `x.y.z` format. Without the last version, package managers think the version is outdated.

Reply:
Oh! How do you verify that warning?